File: OOP Java/lab1/src/main/java/org/example/commands/Cr.java
package org.example.commands;

import org.example.utility.Operands;

import java.util.StringTokenizer;

public class Cr implements Command{
    @Override
    public void Execute(StringTokenizer tokens, Operands operands) {
        System.out.print('\n');
    }

    @Override
    public void Check(StringTokenizer tokens, Operands operands) {

    }

}

File: lab2/Cr.cpp
#include "Cr.h"
#include "FactoryInitializer.h"

void Cr::Execute(Operands& operands, Tokens& tokens, std::string& output, Reader& reader){
    output += "\n";
}

namespace {
    FactoryInitializer<Cr> Registration("cr");
}